src/components/MdxComponents/index.js

```diff
@@ -49,8 +58,15 @@ export default {
   p: (props) => <p className="paragraph" {...props} />,
   pre: (props) => <pre css={pre} {...props} />,
   table: (props) => <table css={table(useTheme())} {...props} />,
-  em: Jargon,
+  em: emphasis,
+  img: (props) => (
+    <a href={props.src} target="_blank" rel="noopener noreferrer">
+      <img loading={'lazy'} {...props} />
```

Comment:
Codacy found an issue: [img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.](https://app.codacy.com/manual/matfilipowicz/BooGi/pullRequest?prid=5801182)